entities/humans_swat.xml

```diff
@@ -0,0 +1,106 @@
+<Entities>
+	<Entity name="Pointman" type="Human" editorAutoHeight="false">
```

Comment:
I would suggest keeping Consistent "SWAT Pointman" so we don't get entities confused